packages/design-system/src/components/tooltip/index.js

```diff
@@ -82,40 +90,43 @@ let lastVisibleDelayedTooltip = null;
  * @param {import('react').Node} props.children The children to be rendered
  * @param {import('react').RefObject<HTMLElement>} props.forceAnchorRef The ref of the anchor where the tooltip will be shown [optional]
  * @param {boolean} props.hasTail Should the tooltip show a tail
- * @param {boolean} props.isMirrored Should the tail placement be mirrored over the y-axis (for Right-to-Left)
  * @param {Function} props.onBlur Blur event callback function
  * @param {Function} props.onFocus Focus event callback function
  * @param {Function} props.onPointerEnter Pointer enter event callback function
  * @param {Function} props.onPointerLeave Pointer leave event callback function
```

Comment:
can these be removed too since they are removed from passed value?

Reply:
Ah yes. Thanks